toolkit/cli/smart-contracts-commands/src/update_governance.rs

```diff
@@ -0,0 +1,40 @@
+use jsonrpsee::http_client::HttpClient;
+use partner_chains_cardano_offchain::{
+	await_tx::FixedDelayRetries, update_governance::run_update_governance,
+};
+use sidechain_domain::{MainchainAddressHash, UtxoId};
+
+use crate::read_private_key_from_file;
+
+#[derive(Clone, Debug, clap::Parser)]
+pub struct UpdateGovernanceCmd {
+	#[clap(flatten)]
+	common_arguments: crate::CommonArguments,
+	/// Governance authority hash to be set.
+	#[arg(long, short = 'g')]
+	new_governance_authority: MainchainAddressHash,
+	/// Path to the Cardano Payment Key file.
+	#[arg(long, short = 'k')]
+	payment_key_file: String,
+	/// Genesis UTXO of the new chain, it will be spent durning initialization. If not set, then one will be selected from UTXOs of the payment key.
+	#[arg(long, short = 'c')]
+	genesis_utxo: UtxoId,
+}
+
+impl UpdateGovernanceCmd {
+	pub async fn execute(self) -> crate::CmdResult<()> {
+		let payment_key = read_private_key_from_file(&self.payment_key_file)?;
```

Comment:
#359 can potentially conflict with this line, only heads up, perhaps I'll merge later.